### DISCUSSION TITLE
Getting errors on patchedDependencies

### DISCUSSION THREAD

**Discussion by Milind-Jamnekar** [typehero/typehero#518] [Q&A] [answered]
![image](https://github.com/bautistaaa/typehero/assets/58872755/bb812533-957f-4d4d-a8aa-897a8e237604)

I am new into open source. so it's my first time doing OS contribution. and then i have encounter this error can someone help me out please. when i tried to remove `patchedDependencies` from `package.json` then dependency get install fine and all things works except some section.

**Comment by jhechtf** [comment]
Looks like you're on windows - is it 11 or 10?

**Reply by Milind-Jamnekar** [reply]
Yup windows 10

**Reply by Milind-Jamnekar** [reply, marked as answer]
nvm i tried to setup on wsl and it worked perfectly. I didn't find the reason behind it. but i install standalone pnpm in my wsl and it work.
